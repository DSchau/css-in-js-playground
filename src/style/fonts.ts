export const SANS_SERIF = {
  fontFamily: 'sans-serif',
  '.wf-active &': {
    fontFamily: `"Montserrat", sans-serif`
  }
};

export const SERIF = {
  fontFamily: 'serif',
  '.wf-active &': {
    fontFamily: `"Bitter", serif`
  }
};
