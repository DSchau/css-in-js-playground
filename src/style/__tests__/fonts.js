import * as FONTS from '../fonts';

test('it contains a SERIF font', () => {
  expect(FONTS.SERIF).toMatchSnapshot();
});

test('it contains a SANS_SERIF font', () => {
  expect(FONTS.SANS_SERIF).toMatchSnapshot();
});
