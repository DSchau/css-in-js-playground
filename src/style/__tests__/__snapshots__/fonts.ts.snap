// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it contains a SANS_SERIF font 1`] = `
Object {
  ".wf-active &": Object {
    "fontFamily": "\\"Montserrat\\", sans-serif",
  },
  "fontFamily": "sans-serif",
}
`;

exports[`it contains a SERIF font 1`] = `
Object {
  ".wf-active &": Object {
    "fontFamily": "\\"Bitter\\", serif",
  },
  "fontFamily": "serif",
}
`;
